examples/src/main/scala/example/interop/cats/ExampleCatsInterop.scala

```diff
@@ -1,47 +1,93 @@
 package example.interop.cats
 
+import caliban.ResponseValue.{ ObjectValue, StreamValue }
 import caliban._
+import caliban.interop.cats.ToEffect
+import caliban.interop.cats.implicits._
+import caliban.interop.fs2.implicits._
 import caliban.schema.Schema.auto._
+import cats.effect.std.{ Console, Dispatcher, Random }
+import cats.effect.syntax.all._
+import cats.effect.{ ExitCode, IO, IOApp, LiftIO }
+import fs2.Stream
+import zio.ZIO
+import zio.interop.catz._
+import zio.stream.ZStream
+import zio.stream.interop.fs2z._
 
-import cats.effect.{ ExitCode, IO, IOApp }
-import cats.effect.std.Dispatcher
-import zio.Runtime
+import scala.concurrent.duration._
 
-object ExampleCatsInterop extends IOApp {
+object ExampleCatsInterop extends IOApp.Simple {
 
-  import caliban.interop.cats.implicits._
+  implicit val zioRuntime: zio.Runtime[Any] = zio.Runtime.default
 
-  implicit val zioRuntime: Runtime[Any] = Runtime.default
+  val console = Console[IO]
+  val random  = Random.javaUtilConcurrentThreadLocalRandom[IO]
 
   case class Number(value: Int)
 
-  case class Queries(numbers: List[Number], randomNumber: IO[Number])
+  case class Queries(numbers: List[Number], randomNumber: IO[Number], moreRandomNumbers: Stream[IO, Number])
 
-  val numbers      = List(1, 2, 3, 4).map(Number)
-  val randomNumber = IO(scala.util.Random.nextInt()).map(Number)
+  case class Subscriptions(numbers: Stream[IO, Number])
 
-  val queries = Queries(numbers, randomNumber)
+  val numbers             = List(1, 2, 3, 4).map(Number)
+  val randomNumber        = IO(scala.util.Random.nextInt()).map(Number)
+  val moreRandomNumbers   = Stream.repeatEval(random.nextInt).take(5).map(Number)
```

Comment:
How about using the same random function in both cases?

Reply:
Oh.. Sure, will change. I overlooked it. Thank you.

Reply:
@ghostdogpr fixed, take another look please. thx.